### ISSUE TITLE
Custom UserResource

### ISSUE THREAD

**Issue by cubancodepath** [phpsa/filament-authentication#2]
I try to change the UserResource  class to modify the tables elements, but always return the default table. 

In the config file put that 
``` php
use App\Filament\Resources\MyUserResource;
// ...
return [
    'resources' => [
        'UserResource' => MyUserResource::class,
        'RoleResource' => \Phpsa\FilamentAuthentication\Resources\RoleResource::class,
       'PermissionResource' => \Phpsa\FilamentAuthentication\Resources\PermissionResource::class,
    ],
    'pages' => [
        'Profile' => \Phpsa\FilamentAuthentication\Pages\Profile::class
    ],
    'user_timezone' => 'Pacific/Auckland',
    'preload_roles' => true
```
And modify the table method, but always get the original table

**Comment by phpsa** [comment]
Thank you - will investigate and reply shortly

**Comment by phpsa** [comment]
Thank you, this has been resolved in V1.0.1
